README.md

```diff
@@ -18,10 +18,16 @@
 
 To get started with RunPod File Uploader, execute the following command:
 
+#### Installation
 ```bash
 curl -sSL https://github.com/kodxana/RunPod-FilleUploader/raw/main/scripts/installer.sh -o installer.sh && chmod +x installer.sh && ./installer.sh
 ```
 
+#### Starting server / application
```

Comment:
I don't think this belongs under the "How to Install" section, should probably be in its own section.